packages/accounts-2fa/2fa-client.js

```diff
@@ -0,0 +1,74 @@
+import { Accounts } from 'meteor/accounts-base';
+
+// Used in the various functions below to handle errors consistently
+const reportError = (error, callback) => {
+  if (callback) {
+    callback(error);
+  } else {
+    throw error;
+  }
+};
+
+/**
+ * @summary Verify if the user has 2FA enabled
+ * @locus Client
+ * @param {Object|String} selector Username, email or custom selector to identify the user.
+ * @param {Function} [callback] Called with a boolean on success that indicates whether the user has
+ *    or not 2FA enabled, or with a single `Error` argument on failure.
+ */
+Accounts.has2faEnabled = (selector, callback) => {
+  Accounts.connection.call('has2faEnabled', selector, callback);
+};
+
+/**
+ * @summary Generates a svg QR code and save secret on user
+ * @locus Client
+ * @param {String} appName Optional. It's the name of your app that will show up when the user scans the QR code.
```

Comment:
if it is really optional usually optional arguments are defined as `[argName]` instead of `argName` if I recall correctly from other apiboxes.

Reply:
I couldn't find an example of this... I saw the use of `[something]` for callbacks, so I assumed it's this way because callbacks are functions. Should I still change it?

Reply:
[x] is for optional args, if here it is not optional, it shouldn't have it.

Reply:
Gotcha